libs/SmartSync/src/com/salesforce/androidsdk/smartsync/target/SyncUpTarget.java

```diff
@@ -232,7 +242,7 @@ public int updateOnServer(SyncManager syncManager, JSONObject record, List<Strin
      * @return true if successful
      * @throws IOException
      */
-    public int updateOnServer(SyncManager syncManager, String objectType, String objectId, Map<String, Object> fields) throws IOException {
+    protected int updateOnServer(SyncManager syncManager, String objectType, String objectId, Map<String, Object> fields) throws IOException {
```

Comment:
Same story as deleteOnServer (used to be called from manager but now is called from updateOnServer(syncManager, record, fieldlist).